ReactNativeClient/lib/services/KeymapService.ts

```diff
@@ -0,0 +1,202 @@
+const BaseService = require('lib/services/BaseService');
+const { Logger } = require('lib/logger.js');
+const { shim } = require('lib/shim.js');
+
+class KeymapService extends BaseService {
+	private keymap: { [command: string]: KeymapItem };
+
+	static modifiers = /^(CommandOrControl|Command|Control|Ctrl|Cmd|CmdOrCtrl|Alt|Option|AltGr|Shift|Super)$/;
+	static keyCodes = /^([0-9A-Z)!@#$%^&*(:+<_>?~{|}";=,\-./`[\\\]']|F1*[1-9]|F10|F2[0-4]|Plus|Space|Tab|Backspace|Delete|Insert|Return|Enter|Up|Down|Left|Right|Home|End|PageUp|PageDown|Escape|Esc|VolumeUp|VolumeDown|VolumeMute|MediaNextTrack|MediaPreviousTrack|MediaStop|MediaPlayPause|PrintScreen)$/;
+
+	static defaultKeymap: KeymapItem[] = [
+		{ accelerator: 'CommandOrControl+N', command: 'newNoteItem', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
```

Comment:
The default seems to be `['darwin', 'win32', 'linux', 'freebsd']` so maybe you could remove the platforms property in that case? If platforms property is missing, assume that it works on all platforms (like for the Setting class).